src/navigation/RootStackNavigator.tsx

```diff
@@ -9,16 +10,58 @@ import SettingsStackNavigator from './SettingsNavigator';
 import SearchScreen from '../screens/SearchScreen';
 import MovieScreen from '../screens/MovieScreen';
 import {DetailedMovie} from '../models/Movie';
+import FullCreditsScreen from '../screens/FullCreditsScreen';
+import {Cast} from '../models/Cast';
+import {Crew} from '../models/Crew';
```

Comment:
Da bismo izbegli ovakve inporte, mozda bi bilo dobro napraviti modul od models foldera.
To znaci da bismo unutar /models foldera imali jedan index.ts fajl koji bi importovao i u isto vreme exportovao sve modele iz razlicitih fajlova unutar /models foldera.